geocatbridge/servers/models/geoserver.py

```diff
@@ -1040,6 +1040,7 @@ def clearWorkspace(self, recreate=True) -> bool:
         # Get database datastores configuration and isolation flag
         db_stores = []
         isolated = False
+        acl_rules = {}
         if recreate:
             url = f"{self.apiUrl}/workspaces/{self.workspace}/datastores.json"
             stores = self.request(url).json()["dataStores"] or {}
```

Comment:
Hi @kannes,

This has nothing to do with your work, but could you perhaps make this line a bit more robust while you are at it? Something like:

```suggestion
            stores = self.request(url).json().get("dataStores", {}) or {}
```